tests/data/fixtures/pyrflx.py

```diff
@@ -179,3 +180,13 @@ def fixture_array_message_package(pyrflx_: pyrflx.PyRFLX) -> pyrflx.Package:
 @pytest.fixture(name="array_message_value")
 def fixture_array_message_value(array_message_package: pyrflx.Package) -> pyrflx.MessageValue:
     return array_message_package["Message"]
+
+
+@pytest.fixture(name="message_size_package", scope="session")
+def fixture_message_size_package(pyrflx_: pyrflx.PyRFLX) -> pyrflx.Package:
+    return pyrflx_["Message_Size"]
+
+
+@pytest.fixture(name="message_size")
+def fixture_message_size(message_size_package: pyrflx.Package) -> pyrflx.MessageValue:
+    return message_size_package["Msg"]
```

Comment:
The fixture should be named `message_size_value` to be consistent to the existing fixtures.

Reply:
Fixed in https://github.com/Componolit/RecordFlux/pull/556/commits/352f3bd761512d1957c2e451c95f4b8e8b0a6126.